index.html

```diff
@@ -43,6 +43,172 @@ <h3>LET'S CONNECT</h3>
+             </h4>
+         </div>
+         <div class="btn">
+             <button type="button" class="btn-1">html</button>
+             <button type="button" class="btn-2">css</button>
+             <button type="button" class="btn-3">javaScript</button>
+         </div>
+         <div class="big-btn">
+             <button type="button" class="btn-4">See Project</button>
+         </div>
+
+        </section>
+        <section class="subsection-four">
+            <img src="images/Snapshoot-Portfolio-4.png" alt="">
+        <div>
+          <h3>
+             Multi-Post Stories
+         </h3>
+        </div>
+     <div class="list-items">
+         <h4>CANOPY</h4>
+         <ul class="ul-list">
+             <li>Back End Dev</li>
+         </ul>
+         <ul class="ul-list">
+             <li>2015</li>
+         </ul>
+     </div>
+     <div class="paragraph">
+         <h4>
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+         </h4>
+     </div>
+     <div class="btn">
+         <button type="button" class="btn-1">html</button>
+         <button type="button" class="btn-2">css</button>
+         <button type="button" class="btn-3">javaScript</button>
+     </div>
+     <div class="big-btn">
+         <button type="button" class="btn-4">See Project</button>
+     </div>
+
+        </section>
+    </section>
+    <section class="about-section">
+        <div>
+            <h2>About Myself</h2>
+        </div>
+        <div>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div>
+            <h3>LET'S CONNECT</h3>
+        </div>
+       <div class="social-icons">
+            <ul>
+                <li><a href="#"><img src="images/Vector2.svg" alt=""></a></li>
+                <li><a href="#"><img src="images/Linkedin-icon.svg" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector3.svg" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector.svg" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector1.svg" alt=""></a></li>
+            </ul>
+        </div>
+        <div class="big-btn">
+            <button type="button" class="btn-5">Get my resume</button>
+        </div>
+        <div class="skills">
+            <h4>Languages</h4>
+        </div>
+        <div class="skills">
+            <img src="images/Frame-1.svg" alt="">
+            <img src="images/Frame-2.svg" alt="">
+            <img src="images/Frame-3.svg" alt="">
+        </div>
+        <div class="skills">
+            <h4>Frameworks</h4>
+        </div>
+        <hr class="skills">
+        <div class="skills">
+            <h4>Skills</h4>
+        </div>
+        <hr class="skills">
```

Comment:
-  [ ] Instead of using image showing `Javascript,HTML,CSS`. You should use the only icons and add the languages by text & design it according to the requirement. and use the `ul & li`  stage for this whole section according to requirement.
- The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
- The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`